### DISCUSSION TITLE
v0.10.10: Stable Release

### DISCUSSION THREAD

**Discussion by Amazinite** [endless-sky/endless-sky#10694] [Announcements]
This is a stable release, focused on fixing bugs and making some other small improvements.
* The biggest change this release is the removal of the "hold fire" command, added in 0.10.9, because of bugs associated with it that could not be adequately resolved prior to this release.
* We hope to have addressed those issues for the next release in a few months.

This release brings one new feature:
* The addition of the "silent jumps" and "silent scans" attributes, which allow ships to jump and scan without playing any sounds, not even the default ones.

Some other changes are:
* Particularly large capture defense values on some outfits were reduced, to work better with changes to capturing mechanics in 0.10.9.
* Fixes and improvements to various Incipias, Remnant, and Successor missions.
* Updated landing messages for some star sprites to improve accuracy.

For a complete list of changes, as well as further detail about the summarized changes, see the [changelog](https://github.com/endless-sky/endless-sky/blob/master/changelog). A special thanks to everyone who contributed to this release: @alextd @Anarchist2 @Arachi-Lover @chrisandreae @Daeridanii1 @Hurleveur @Koranir @mOctave @opusforlife2 @Quantumshark @reticent-rem @Saugia @tekker2234 @TheGiraffe3 @thewierdnut @tibetiroka @TomGoodIdea @Zitchas

If you are using the high-dpi plugin, you should download and install the [updated version](https://github.com/endless-sky/endless-sky-high-dpi/releases/tag/v0.10.10) that goes with this release.

![splinter_successor_space](https://github.com/user-attachments/assets/fab982ef-fb7c-4c09-9c7f-33f0a2f423f4)

<hr /><em>This discussion was created from the release <a href='https://github.com/endless-sky/endless-sky/releases/tag/v0.10.10'>v0.10.10: Stable Release</a>.</em>

**Comment by ketronix-dev** [comment]
I still waiting for some features for translators. Cuz i can help with localization, but i dont see any changes on that way :(

**Comment by esksp** [comment]
Is this compatible with 0.10.8 savegames?

**Reply by danaris** [reply]
Yes. All new releases are backward-compatible with old saved games.

**Comment by Elisson-cyber** [comment]
Please dont change the directx to 10 or above im playing you game on a cell phone android with some tool kkkkk thanks for the excelent game keep going to the future.....hug from brazil

**Reply by Koranir** [reply]
We don't use DirectX, only OpenGL 3, so you should be fine.

There's a dedicated android fork made by theweirdnut, so try that out if you want to.

**Comment by Zeesh4n-Ali** [comment]
I am using windows 7 32bit , which file do i download?

**Reply by TheGiraffe3** [reply]
There's been a new release since then: https://github.com/endless-sky/endless-sky/releases/tag/v0.10.11
Do you know how to build the application?

**Reply by Zeesh4n-Ali** [reply]
Build application,,,? No. And what file to download

**Reply by Zeesh4n-Ali** [reply]
There are 3 files? Which one do i download

**Reply by TheGiraffe3** [reply]
You could try the win-64 one. If that doesn't work I think you're stuck building it.

**Reply by TomGoodIdea** [reply]
There are no official 32-bit builds anymore. If you want to play the latest version, you need to compile it yourself or change your OS.

**Reply by Zeesh4n-Ali** [reply]
Meaning i can't play the game, welp there goes my plans for modding :(

**Reply by Zeesh4n-Ali** [reply]
What do you mean by building?

**Reply by warp-core** [reply]
https://github.com/warp-core/endless-sky/actions/runs/12990435220?pr=52
Scroll down to the bottom of this page and download the file with "win32" in its name and try that. I think it should run on your system.

**Reply by Zeesh4n-Ali** [reply]
Wait a sec, i don't understand, is that official, what version is that? Did you build something for me , what is going on?

**Reply by TheGiraffe3** [reply]
That is the latest 0.10.11 version.
It's not official yet, but it may end up being.
He built it for you, yes (and it is 32-bit, so should work fine for you).
[This](https://productionresultssa9.blob.core.windows.net/actions-results/8fee0122-9075-41d9-8d63-ad7ffd5822a1/workflow-job-run-6a0a0f03-6a8e-569e-614a-9d2fe942be76/artifacts/36f742d0e33f6de6fdde05ff006fa847fe1a2bf95f504c914f29c95915d284d3.zip?rscd=attachment%3B+filename%3D%22EndlessSky-win32-continuous.zip.zip%22&se=2025-01-28T05%3A19%3A13Z&sig=7iameRqgJSyCgZRAzWQniSfTDv6aIuX6BN5ZvUMb%2BlQ%3D&ske=2025-01-28T13%3A39%3A50Z&skoid=ca7593d4-ee42-46cd-af88-8b886a2f84eb&sks=b&skt=2025-01-28T01%3A39%3A50Z&sktid=398a6654-997b-47e9-b12b-9515b896b4de&skv=2025-01-05&sp=r&spr=https&sr=b&st=2025-01-28T05%3A09%3A08Z&sv=2025-01-05) is the direct download link.

**Reply by esksp** [reply]
> What do you mean by building?

Sorry, you were speaking to developers, and they have a very special kind of language...
FYI, "building" a piece of software means downloading the source code and [compiling](https://en.wikipedia.org/wiki/Compiler) it to run on your computer. Given you don't even know that term, this would be definitely _way_ out of your reach (mine too, BTW)... 😆 
Fortunately, despite being developers they are very nice people, so they did it for you... 😛

**Reply by Zeesh4n-Ali** [reply]
Yeah, these guys are awesome, ikr now that it is not supported , if they don't release the 32bit stable release in future, i will learn to do that, it would be a new thing to learn and i love learning new things, i am again very thankful to the kind people like these, no wonder why indie games are so good, i wish more people find out about this amazing project.

Thanks again.

**Comment by Zeesh4n-Ali** [comment]
So someone built the 32 bit for me, but it was the unstable ver, so it's crashing it said it's the continous build idk what's that , so I'm asking if the fiture stable updates , they'll bring back 32bit support or I'm stuck with 0.10.2 version.

**Reply by Zitchas** [reply]
Stable vs unstable vs continuous is nothing more than a label indicative of the development process. 
Continuous = The project released as of right now. (a new continuous is available after ever merge of new stuff into the project)
Unstable = A release made at the end of a period of focusing on adding new content & mechanics (often includes fixes too)
Stable = A release made at the end of a period where the focus was on fixes (often includes minor content and mechanics too, but nothing big)

**Reply by Zeesh4n-Ali** [reply]
Yeah but the continous one just straight up freezes my pc, and that's the only version someone built bor me for 32bit

**Reply by warp-core** [reply]
The package I built uses the same code as the 0.10.11 release.
Given that, if we add an "official" 32-bit release, the procedure to build it will be (almost) identical to what I used to make the package I linked, I'm not sure that that's going to help you.
The only causes of crashes I'm currently aware of are:
- insufficient memory. Most people only encounter this with the high DPI plugin installed. I'm assuming you didn't add that, but if you did, I suggest removing it and trying again. If the issue is present even without high DPI and memory capacity is the issue, there is one or two other things we can try.
- a bug in the compiler that causes problems if the game is located in a path with some unicode (non-ascii) characters in it. The 64-bit version is built with a slightly newer version of the compiler that doesn't appear to have this issue, but I cannot currently (easily) get that version for building a 32-bit package in GitHub Actions. I can build it locally with the newer compiler which should quickly show us if that's your issue, but you probably shouldn't run something I build on my computer and then send to you. Can you share hte path where you've installed Endless Sky?

Can you provide more details about what happens? Does the Endless Sky window pop up at all? Can you open task manager and monitor memory usage from the "performance" tab (use the "more details" button at the bottom left if you don't see any tabs) while you launch the game?

We can also try building each release version since the last one we did in 32-bit. I expect the procedure I used for 0.10.11 will also work with all of those, although most of the increase in memory usage has been since 0.10.8, so we can probably start there and see if that works.

**Reply by warp-core** [reply]
https://github.com/warp-core/endless-sky/actions/runs/13010461762

The packages here are built with the exact same process that is likely to be used for an "official" release. The only reason to make changes would be if this doesn't work, and I cannot currently say what those changes might be. It is almost the same as what I did before, but not quite.

You can try the "[EndlessSky-win32-Win32-Release-CD-Test-II.zip](https://github.com/warp-core/endless-sky/actions/runs/13010461762/artifacts/2497164434)" file, but I doubt there will be much difference.

**Reply by Zeesh4n-Ali** [reply]
The game pops up and at the starting screen where the star loads, it just freezes everything and i have to cut off the power and restart, tried a couple times and faced same fate, then tried the old official version and it worked perfectly, i tried the 64bit ver of that continous build in my school lab and guess what, it faced the same issue , stuck at 80 percent or so , and when i restart the oc , it takes extra time to boot up, i almost broke lab pc and mine so i just deleted them , it would be better if you make 32 bit for this version (0.10.10) and if possible, in future or else , just thanks for your efforts, I'll need to be at the old version.

**Reply by warp-core** [reply]
This: https://github.com/warp-core/endless-sky/actions/runs/13072893427
will build 0.10.8 for 32-bit.

This: https://github.com/warp-core/endless-sky/actions/runs/13072934420
will build 0.10.10 for 32-bit.

They might take as many as 40 minutes to finish, or as few as 10.
Let me know if either of them work for you.

**Reply by warp-core** [reply]
It looks like those aren't working. I'll try to get them sorted and get back to you.

**Reply by warp-core** [reply]
In the meantime, you could try manually setting the "reduce large graphics" preference which may enable you to run 0.10.11 on your system.
If you are able to launch v0.10.2, you can do this from the settings.
After closing that, you can try launching v0.10.11 again.

**Reply by warp-core** [reply]
It looks like this build of 0.10.10 has a worked. https://github.com/warp-core/endless-sky/actions/runs/13073482172

**Reply by Zeesh4n-Ali** [reply]
Is there a way to update? Just move older to trash and extract new?? Or more intuitive approach? That saves get preserved?

**Reply by warp-core** [reply]
Deleting all the old game files an extracting the new ones is the best way (alternatively, you could extract the new files into a different folder).
Saves are stored in your "appdata" folder. Unless you installed the game files in that folder, which is unlikely, your saves will all still be there even if you delete the game files.

**Reply by warp-core** [reply]
@Zeesh4n-Ali were you able to get 0.10.10 working?
